shared/api/buildChatMessages.ts

```diff
@@ -0,0 +1,15 @@
+import ChatMessageModel from './models/ChatMessageModel'
```

Comment:
I am not sure whether it is okay to split this into a separate helper function...

Reply:
🔧 Very much okay to move this to a separate helper function, however, in order to align with our current naming, this should be called `mapChatMessages` and reside in `shared/api/mapping/mapChatMessages`.